dependencies.yaml

```diff
@@ -178,7 +169,40 @@ dependencies:
               cuda: "12.*"
             packages:
               - cuda-cudart-dev
+      - output_types: conda
+        matrices:
+          - matrix:
+              arch: x86_64
+              cuda: "11.8"
+            packages:
+              - libnvjpeg=11.6.0.55
+              - libnvjpeg-dev=11.6.0.55
+          - matrix:
+              arch: aarch64
+              cuda: "11.8"
+            packages:
+              - libnvjpeg=11.6.0.55
+              - libnvjpeg-dev=11.6.0.55
```

Comment:
Maybe I'm missing it. Is there a difference between these two cases or can they be consolidated?

Reply:
These can be consolidated! I just copied from before, where they were split. Done in 057200c.

Reply:
Ah ok. Sorry we were overly verbose before